README.md

```diff
@@ -1,22 +1,45 @@
 # suspend-time
-A cross-platform monotonic clock that is suspend-unaware, written in Rust!
+Suspend-time is a cross-platform monotonic clock that is suspend-unaware, written in Rust!    
+It allows system suspension (e.g. when a user closes their laptop on windows) to not affect `Instant` durations and timeouts!
 
-**Documentation**: [to be hosted]
+**[API Documentation](https://docs.rs/suspend-time/latest/suspend_time/)**
 
 ## Example
 
+Example of using `SuspendUnawareInstant`:
+
 ```rust
 use std::{thread, time};
 use suspend_time::{SuspendUnawareInstant};
 
 fn main() {
+    // If you used std::time::Instant here and you suspend the system on windows,
+    // it will print that more than 3 seconds (circa July 2024).
+    // With SuspendUnawareInstant this has no effect.
     let instant = SuspendUnawareInstant::now();
     let three_secs = time::Duration::from_secs(3);
     thread::sleep(three_secs);
-    assert!(instant.elapsed() >= three_secs);
+    println!("{:#?}", instant.elapsed());
 }
 ```
 
+Example of using `suspend_time::timeout`:
```

Comment:
Maybe you want to just reference `examples/timeout.rs` here?

Reply:
It's nice to see in the README.md -- I am copying a pattern from other OSS projects I like.